x/concentrated-liquidity/incentives_test.go

```diff
@@ -3534,6 +3545,25 @@ func (s *KeeperTestSuite) TestQueryAndClaimAllIncentives() {
 				}
 				s.Require().Equal(expectedCoins, amountClaimed)
 				s.Require().Equal(sdk.Coins(nil), amountForfeited)
+
+				uptimeAccumsNew, err := clKeeper.GetUptimeAccumulators(s.Ctx, validPoolId)
+				s.Require().NoError(err)
+
+				var newAccumRecord []accum.Record
+				for _, uptimeAccum := range uptimeAccumsNew {
+					positionName := string(types.KeyPositionId(DefaultPositionId))
+					accRecord, err := uptimeAccum.GetPosition(positionName)
+					s.Require().NoError(err)
+
+					newAccumRecord = append(newAccumRecord, accRecord)
+				}
+				expectedUptimeAccValue := sdk.NewDecCoinsFromCoins(sdk.NewCoin("bar", sdk.NewInt(100)), sdk.NewCoin("foo", sdk.NewInt(100)))
```

Comment:
Where are these values coming from? Why is it 100 of each coin?

Let's avoid hardcoding, even the denoms